subcommittee/tooling/meetings/07-February-2025/minutes.md

```diff
@@ -0,0 +1,60 @@
+    - will be addressed by the coding guidelines subcommittee
+
+Aerospace DO-178
+Automotive ISO26262
+
+## Open topics
+- To be aligned to Coding Guidelines (CG)
+- Do we need static analysis for DAL-D and DAL-E?
+- *Format Verification* needs to be refined at some point
+- *Code Traceability* needs to be refined for DO-178
+
+### Initial proposal
+
+This table contains the inital proposal that we had before the meeting and that we did not copy to the output table.
+
+| Purpose | Requirement | Tool | Status |
+|---------|-------------|------|--------|
+| Qualified Compiler | ? | [Ferrocene](https://ferrocene.dev/en/) | Available for `aarch64-nostd` |
+| Certified Core Library | SIL-4 / ASIL-D out of context | N/A | In progress by Ferrocene / Adacore / HighTec |
+| Coding Style Verification | MISRA-C, ... | Rust Compiler, Clippy and probably additional tools required to verify and evaluate the application of the coding standard developed by the Code Guidelines Subcommittee | OxidOS can provide a mapping of MISRA-C Rules to the Rust Compiler |
+| Static Analysis Tools | probably similar to Polyspace | N/A | an assesement about what the rust compiler covers is necessary |
+|Unambiguous graphical representation|ISO26262 ASIL B/C/D | N/A | Graphical representation specific not available. Autosar Rust defined a Enterprise Architect profile See chapter 6.6 of [Explanation of ARA Applications in Rust](https://www.autosar.org/fileadmin/standards/R23-11/AP/AUTOSAR_AP_EXP_ARARustApplications.pdf)|
+|Use of naming convention| ISO26262 ASIL A/B/C/D | Clippy | Not available tool that allow to define and enforce naming convention. Clippy check only for predefined and generic naming convention |
+|Measurement of execution time and reaction time of software unit| ISO 26262 6.4.2 e) | perf, Intel VTune, flamegraph, [Rapita](https://www.adacore.com/press/rapita-systems-showcases-adacores-gnat-pro-for-rust-at-hisc) |  For hard realtime application are required not intrusive profiler that are typically HW specific |
+| Test runner | all | cargo-nextest, cargo, defmt-test+probe.rs | No ISO qualified tool are available. Example of qualified test runner: [VectorCast](https://www.vector.com/us/en/products/products-a-z/software/vectorcast/?gad_source=1&gclid=EAIaIQobChMIyMKCgvn3igMVM0-RBR1s9BOkEAAYASAAEgJM2_D_BwE#c336977),  [TESSY](https://www.razorcat.com/en/product-tessy.html)|
+| Automatic generation of unit/integration tests based on equivalence classes and boundary values | ISO26262 Part 6 (9.4.4 & 10.4.4) ASIL B/C/D | N/A | Example of automatic test generator based on equivalence classe and boundary values: [VectorCast](https://www.vector.com/us/en/products/products-a-z/software/vectorcast/?gad_source=1&gclid=EAIaIQobChMIyMKCgvn3igMVM0-RBR1s9BOkEAAYASAAEgJM2_D_BwE#c336977),  [TESSY](https://www.razorcat.com/en/product-tessy.html) |
+
+
+## Next Meeting
+
+At Rust Nation - 19.02.2025
```

Comment:
Did the tooling team actually meet at Rust Nation? Or are you just saying members of the tooling committee will be at the full consortium meeting?

Reply:
We were supposed to meet, but we never did the breakouts. I think it is safe to say we did meet.